src/components/HubSpotForm/index.js

```diff
@@ -62,24 +62,33 @@ export class HubSpotForm extends React.Component {
   };
 
   render() {
-    const { className, title, description } = this.props;
+    const { className, title, description, isLight } = this.props;
     const { isLoaded } = this.state;
 
+    const hubspotForm = isLight ? (
+      <div
+        id="hubspot-form"
+        className={cn('mx-auto', className)}
+        style={{ display: isLoaded ? 'block' : 'none', maxWidth: 600 }}
+      />
+    ) : (
+      <div
+        id="hubspot-form"
+        className={cn('bg-white shadow rounded p-16 md:p-4 mx-auto md:overflow-hidden', className)}
+        style={{ display: isLoaded ? 'block' : 'none', maxWidth: 600 }}
+      />
+    );
+
     return (
       <div className="w-full">
-        <h2 className="text-3xl text-center">{title}</h2>
-        <div className="flex justify-center flex-wrap items-center pb-12 md:pb-12">
+        {title && <h2 className="text-3xl text-center">{title}</h2>}
+        {description && <div className="flex justify-center flex-wrap items-center pb-12 md:pb-12">
           <div
             className="font-default opacity-75 text-xl max-w-lg mt-4 md:mt-6 mx-auto text-center"
             dangerouslySetInnerHTML={{ __html: description }}
           />
-        </div>
-
-        <div
-          id="hubspot-form"
-          className={cn('bg-white shadow rounded p-16 md:p-4 mx-auto md:overflow-hidden', className)}
-          style={{ display: isLoaded ? 'block' : 'none', maxWidth: 600 }}
-        />
+        </div>}
+        {hubspotForm}
```

Comment:
Since `className` is the only thing that changes with `isLight`, I suggest just keeping the `div` inline and setting the classname using the `cn` util like the following:

```suggestion
        <div
            id="hubspot-form"
            className={cn('mx-auto', className, {
              'bg-white shadow rounded p-16 md:p-4 md:overflow-hidden': !isLight
            }
            style={{ display: isLoaded ? 'block' : 'none', maxWidth: 600 }}
         />
```